src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
if it is visual effect, than I can think one of case where user would take advantage of it. user puts warning and then for warnings that they provide code fix (if they provide both diagnostics, not individual diagnostic, but diagnostic ID and fixes for them, then they will know which their diagnostic ID has fix and which one doesn't have), they can argument it with suggestion tag. 

making their warning that has code fix to have different appearance than ones that don't have.